docs/content/stable/architecture/docdb-sharding/cluster-balancing.md

```diff
@@ -0,0 +1,234 @@
+
+With yugabyted, you can access the YB-Master UI at <http://localhost:7000> to access two views to monitor the cluster balancer progress.
+
+#### Tablet Servers view
+
+The "Tablet Servers" view contains information about how many [peers](../../docdb-replication/replication/#tablet-peers) and [leaders](../../key-concepts/#tablet-leader) are hosted on each tablet server, as well as whether each tablet server is currently data or leader blacklisted.
+
+To access the view, click **Tablet Servers** on your YB-Master UI.
+
+#### Cluster Balancer view
+
+The Cluster Balancer view displays information about:
+
+- Ongoing cluster balancing tasks (for example, [tablet leader](../../key-concepts/#tablet-leader)ship movement, [tablets](../../key-concepts/#tablet) being added or removed).
+- Any warnings during cluster balancing (for example, a placement policy being unsatisfiable because there are no TServers in the requested zone)
+- Ongoing remote bootstraps from one tablet server to another, including the speed of data transfer.
+
+To access the view, click **Utilities > Load Balancer** on your YB-Master UI.
+
+  </div>
+  <div id="yba" class="tab-pane fade" role="tabpanel" aria-labelledby="yba-tab">
+
+The [YugabyteDB Anywhere](../../../yugabyte-platform/) [Metrics](../../../yugabyte-platform/alerts-monitoring/anywhere-metrics/) page has graphs to monitoring cluster balancing progress.
+
+You access metrics by navigating to **Universes > Universe-Name > Metrics**.
+
+#### Data Disk Usage graph
+
+The "Data Disk Usage" graph displays the amount of disk space used by the selected tablet server. The disk usage should be similar on all tablet servers after data is balanced.
+
+To view the graph, select the **Resources** tab on your universe **Metrics** page.
+
+#### Cluster load balancer statistics graph
+
+The "Cluster load balancer statistics" graph displays an estimate of the number of [tablets](../../key-concepts/#tablet) that still have to be moved before the cluster is considered balanced.
+
+To view the graph, select the **Master Server** tab on your universe **Metrics** page as per the following illustration:
+
+![Cluster load balancer statistics](/images/architecture/cluster-balancer-graph.png)
+
+Cluster balancing is complete when this graph reaches 0.
+
+#### Monitoring progress from metrics
+
+The graph above shows the following metrics, which can be used to track the number of [tablet](../../key-concepts/#tablet) moves that are required before the cluster is considered balanced:
+
+- `total_table_load_difference`
+- `blacklisted_leaders`
+- `tablets_in_wrong_placement`
+
+The metric `estimated_data_to_balance_bytes` tracks how much data (in bytes) must be transferred before a [cluster](../../key-concepts/#cluster) is balanced. This is often the best indicator of how long cluster balancing will take, as many clusters are disk or network limited.
```

Comment:
This is the most accurate indicator of how long cluster balancing will take for clusters that are limited by data transfer speed